tests/store/test_store_models.py

```diff
@@ -1,28 +1,33 @@
-# -*- coding: utf-8 -*-
-from pathlib import Path
-import datetime
+"""Tests for the models"""
 
 from housekeeper.store import models
 
 
-def test_Version():
-    # GIVEN a bundle version
-    bundle_obj = models.Bundle(name='handsomepig')
-    now = datetime.datetime.now()
-    version_obj = models.Version(created_at=now, bundle=bundle_obj)
-    # WHEN accessing the relative root path
-    root_dir = version_obj.relative_root_dir
-    # THEN it should point to the correct folder
-    now_str = str(now.date())
-    assert root_dir == Path('handsomepig') / now_str
+def test_instantiate_bundle(case_id, timestamp):
+    """Test instantiate a Bundle object"""
+    # GIVEN a case id and a time stamp
+    # WHEN instantiating a bundle
+    bundle_obj = models.Bundle(name=case_id, created_at=timestamp)
+    # THEN assert it was instantiated as expected
+    assert bundle_obj.name == case_id
+    assert bundle_obj.created_at == timestamp
+
+
+def test_instantiate_bundle_no_name():
+    """Test instantiate a Bundle object without a name"""
+    # WHEN instantiating a bundle without a name
+    bundle_obj = models.Bundle()
+    # THEN assert it was instantiated as expected
+    assert bundle_obj.name is None
```

Comment:
What happens here? It this a valid bundle object? No version and no name? Or shoudl the code log and error?

Reply:
It just tests the current behaviour of the model. If we want to change the model we can change the test